sklearn/feature_extraction/tests/test_image.py

```diff
@@ -39,6 +39,14 @@ def test_grid_to_graph():
     A = grid_to_graph(n_x=size, n_y=size, mask=mask, return_as=np.ndarray)
     assert connected_components(A)[0] == 2
 
+    # check ordering
```

Comment:
Could you make a similar test using the function `img_to_graph` to be sure that in case we have weights, the weights are properly picked.